lib/common/util.js

```diff
@@ -160,6 +160,18 @@ exports.timestamp = function (aDate) {
 };
 
 
+/*
+ * Returns the timestamp corresponding to the date "date".
+ * "date" can be a string or a number.
+ */
+function convertDateStringToNumeric(date) {
```

Comment:
Could this just be `parseDateTime()` from [jsprim][2]?

[2]: https://github.com/davepacheco/node-jsprim#parsedatetimestr